CHANGELOG.md

```diff
@@ -57,5 +61,5 @@
 
 - Initial version ([@DmitryTsepelev][])
 
-[@DmitryTsepelev]: https://github.com/DmitryTsepelev
+[@dmitrytsepelev]: https://github.com/DmitryTsepelev
```

Comment:
please switch this line back

Reply:
weird, looks like one of my linters tweaked this - reverted via https://github.com/DmitryTsepelev/store_model/pull/38/commits/a8d9cb3d437b6aedbbd88c6027227d9885c46ad0.